qcodes/instrument_drivers/Keysight/Infiniium.py

```diff
@@ -366,6 +367,7 @@ class Infiniium(VisaInstrument):
 
     def __init__(self, name: str, address: str,
                  timeout: float = 20,
+                 acquire_mode: str = 'RTIMe',
```

Comment:
I don't see the need to actually pass this to init function of the scope since it is already a parameter on the scope. If you want to set it to a default value (which I don't think we should) one can simply pass that as initial_value to `add_parameter`

Reply:
Yes, you are right. We do not need to pass this value to init function. But since `RawTrace` class had previously set `acquire_mode` parameter to this default value, we do need to set it as initial_value. Is there better way to handle it?

Reply:
I am not sure if we need that. It will be a breaking change no matter what. If we set the initial value then users may be surprised that the mode changes when they connect to the scope and setting it once on init will anyway not match the previous behavior

Reply:
@jenshnielsen previously the acquire mode was set to a hard-coded value at acquisition time, which I think is equally surprising especially if you set the value manually and it then changes when you connect via qcodes. Perhaps this acquire mode should just default to None and only be set in the constructor if the user specifies it?

Reply:
> @jenshnielsen previously the acquire mode was set to a hard-coded value at acquisition time, which I think is equally surprising especially if you set the value manually and it then changes when you connect via qcodes. Perhaps this acquire mode should just default to None and only be set in the constructor if the user specifies it?

Sorry I think you are misunderstanding what the issue at discussion is now. 
We all agree that this should not be set as part of `RawTrace` `get_raw` method. Neither should it be set as part of `prepare_curve_data` 

The problem is if it should be part of the constructor of the Infiniium class at all. I am arguing that it should **not** since that would allow it to change the value when connecting to the instrument which is confusing and error prone and not how most qcodes drivers work. Furthermore it is not needed at all since it is already a parameter on the instrument and the user can simply set it as such. 

```
inst = Infiniium(name='name', addr='addr)
inst.acquisition_mode('somemode') 
```

Reply:
@jenshnielsen , @guenp has let me know that to run this driver, they just comment out the `acquire_mode` call in the `RawTrace` `get_raw` method without any other change to the driver. In this case, I think if we remove `acquire_mode` call from all places (both `RawTrace` and Infiniium constructor) we should be fine. I will implement this now and will test this with help of @guenp and Rachpon next to next week when I am back.